### PR TITLE
Add operations related to per-domain SSO.

### PR DESCRIPTION
**Pull request by thetorpedodog**
These will allow organization administrators to set up per-domain SSO for their organization and connect it to their domain.

---

This should account for all the operations we need; I’m going to keep it open until the implementation is there, though.

### CODE REVIEW

File: openapi-v1.yaml [hunk 1]
```diff
@@ -5487,6 +5591,113 @@ paths:
...
+            $ref: '#/definitions/SSODomainConfig'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+    patch:
+      tags:
+        - organization
+      description: Updates the configuration for the given SSO connection.
+      operationId: updateSSODomain
```
Comment (alekshs) [inline openapi-v1.yaml:5678]: The PATCH body is missing. Should we add the SSODomainConfig ref or a new definition?

**Review by alekshs — CHANGES_REQUESTED** [review]

File: openapi-v1.yaml (hunk 1 above)
Comment (thetorpedodog) [reply openapi-v1.yaml:5687]: that seems important

**Review by alekshs — APPROVED** [review]

File: openapi-v1.yaml [hunk 2]
```diff
@@ -5489,6 +5593,146 @@ paths:
...
+        type: string
+    post:
+      tags:
+        - organization
+      description: >
+        Immediately verify ownership of the specified SSO domain ownership
+        claim.
+      operationId: checkSSODomain
+      responses:
+        200:
```
Comment (SarantopoulosKon) [inline openapi-v1.yaml:5730]: We are missing the error for the `run_check`

```
        default:
          description: error response
          schema:
            $ref: '#/definitions/Error'
```

**Review by SarantopoulosKon — DISMISSED** [review]

File: openapi-v1.yaml (hunk 2 above)
Comment (thetorpedodog) [reply openapi-v1.yaml:6251]: haha holy shit I forgot about this change for literal months. fixed.

**Review by SarantopoulosKon — APPROVED** [review]

**Pull request merged**